modules/extension/wms/src/test/resources/org/geotools/ows/wms/map/caps130.xml

```diff
@@ -23,6 +23,7 @@
       </GetCapabilities>
       <GetMap>
         <Format>image/png</Format>
```

Comment:
What will happen if `ìmage/png` is not supported by the cascaded server?

Reply:
the current implementation assumes that the cascaded server will provide image/png as a bare mimimum. With the currently available constructor is if `image/png` is not supported then the WMS requests will probably go incomplete, shall I throw an Exception for this?

Reply:
The previous implementation expects `ìmage/png` as bare minimum to be available, thats why it always defaults to `image/png`